src/lib/rule-finder.js

```diff
@@ -72,13 +72,16 @@ function _getPluginRules(config) {
 function _getAllAvailableRules(pluginRules) {
   var allRules = fs
     .readdirSync('./node_modules/eslint/lib/rules')
-    .map(function removeJsFromFilename(filename) {
-      return filename.replace(/\.js$/, '')
-    })
+  var filteredAllRules = []
+  allRules.forEach(function filterAllRules(filename) {
```

Comment:
Solely because it's ideologically cleaner/pure/more functional than a forEach + a closure. not a huge deal.